src/rust/engine/options/src/lib.rs

```diff
@@ -522,16 +560,15 @@ impl OptionParser {
         id: &OptionId,
         default: Vec<T>,
         getter: fn(&Arc<dyn OptionsSource>, &OptionId) -> Result<Option<Vec<ListEdit<T>>>, String>,
-        remover: fn(&mut Vec<T>, &Vec<T>),
+        remover: fn(&mut Vec<T>, &[T]),
```

Comment:
This was a clippy lint that somehow didn't come up before (maybe related to the fact that we now use this outside this file).